files/zh-tw/web/api/setinterval/index.md

```diff
@@ -475,7 +475,9 @@ MiniDaemon.prototype.start = function (bReverse) {
 
 ### Syntax
 
-` var myDaemon = new MiniDaemon(thisObject, callback[``, rate``[, length]]); `
+```
```

Comment:
````suggestion
```js
````

Reply:
These syntax blocks don't get the language code because they're not really proper JS. They've been using `js-nolint` as a workaround for some of them upstream, but I'm not sure if they've landed on that for everything

Reply:
let's ignore this suggestion for now